src/Build/Construction/ProjectSdkElement.cs

```diff
@@ -35,45 +52,46 @@ internal ProjectSdkElement(ProjectElementContainerLink link)
         /// </summary>
         private ProjectSdkElement(XmlElementWithLocation xmlElement, ProjectRootElement containingProject)
             : base(xmlElement, null, containingProject)
-        { }
+        {
+        }
 
         /// <summary>
         /// Gets or sets the name of the SDK.
         /// </summary>
         public string Name
         {
-            get => GetAttributeValue(XMakeAttributes.sdkName);
+            get => GetAttributeValue(XMakeAttributes.sdkName, true);
```

Comment:
I've decided to annotate this property as non-nullable, but thanks to `GetAttributeValue` not being NRT-annotated, I don't even need to put a `!` here.